include/picongpu/particles/atomicPhysics/enums/ChooseTransitionGroupFor.hpp

```diff
@@ -45,12 +45,7 @@ namespace picongpu::particles::atomicPhysics::enums
         static constexpr ChooseTransitionGroup chooseTransitionGroup = ChooseTransitionGroup::boundBoundDownward;
     };
 
-    //! bound-free(upward)
-    template<>
-    struct ChooseTransitionGroupFor<TransitionType::boundFree, TransitionDirection::upward>
-    {
-        static constexpr ChooseTransitionGroup chooseTransitionGroup = ChooseTransitionGroup::boundFreeUpward;
-    };
+    //! bound-free(upward) may be either collisionalBoundFreeUpward or fieldBoundFreeUpward -> error to ask
```

Comment:
what does `error to ask` mean?

Reply:
discussed offline, moved comment to file documentation and extended the file description to make it better understandable what the intention of the file is.